pkgs/development/tools/vultr/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "vultr-${version}";
+  version = "${major}.${minor}.${patch}";
+  major = "1";
+  minor = "12";
+  patch = "0";
```

Comment:
OK, I adopted this from https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/doctl/default.nix#L5 and thought it was best practice, but I'll change it.